api/bin/api_token.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Command line utility for creating an API token
+ *
+ * Usage:
+ *  npm run exec ./bin/api_token.ts -- --cb=CB_NAME --scope=SCOPE
```

Comment:
> would it be easier to use cb id here so that there's no spaces?

You can just use quote marks when specifying the argument? IMO it's 6 of one, because either you use quote marks or you have to lookup the ID.

Reply:
Okay, can we check that the cb name exists before adding?

Reply:
Yup, should have been added in 54883d9